docs/messages.md

```diff
@@ -5,6 +6,42 @@ description: A superclass for the messages of the bitcoin network
 
 The bitcoin protocol specifies a set of [messages](https://en.bitcoin.it/wiki/Protocol_specification) that can be sent from peer to peer. `bitcore-p2p` provides support for some of these messages.
 
+To create a messages, you can use any of the message constructors, here is a simple example:
```

Comment:
Fixed in: https://github.com/bitpay/bitcore-p2p/commit/664ceb2d30e79dbe98057dc1dd131f317573b47a